File: GrowthWare_CS/GrowthWareFramework/ModelObjects/MChangePasswordFormatter.cs

using System;
using System.Collections;
using System.Collections.Generic;
using System.Data;
using System.Diagnostics;
using System.Web;
using GrowthWare.Framework.ModelObjects;
using GrowthWare.Framework.ModelObjects.Base;

namespace GrowthWare.Framework.ModelObjects
{
	[Serializable(), CLSCompliant(true)]
	public class MChangePasswordFormatter : MMessageProfile
	{
#region "Member Properties"
		private string m_AccountName = string.Empty;
		private string m_FullName = string.Empty;
		private string m_Password = string.Empty;
		private string m_Server = string.Empty;
#endregion

#region "Public Properties"
		public string AccountName {
			get { return m_AccountName; }
			set { m_AccountName = value.Trim(); }
		}

		public string FullName {
			get { return m_FullName; }
			set { m_FullName = value.Trim(); }
		}

		public string Password {
			get { return m_Password; }
			set { m_Password = value.Trim(); }
		}

		public string Server {
			get { return m_Server; }
			set { m_Server = value.Trim(); }
		}
#endregion
	}
}

File: GrowthWare_VB/GrowthWareFrameworkModel/Profiles/MChangePasswordFormatter.vb
Imports GrowthWare.Framework.ModelObjects
Imports GrowthWare.Framework.ModelObjects.Base

Namespace ModelObjects
	<Serializable(), CLSCompliant(True)> _
	Public Class MChangePasswordFormatter
		Inherits MMessageProfile

#Region "Member Properties"
		Private _AccountName As String = String.Empty
		Private _FullName As String = String.Empty
		Private _Password As String = String.Empty
		Private _Server As String = String.Empty
#End Region

#Region "Public Properties"
		Public Property AccountName() As String
			Get
				Return _AccountName
			End Get
			Set(ByVal value As String)
				_AccountName = value.Trim
			End Set
		End Property

		Public Property FullName() As String
			Get
				Return _FullName
			End Get
			Set(ByVal value As String)
				_FullName = value.Trim
			End Set
		End Property

		Public Property Password() As String
			Get
				Return _Password
			End Get
			Set(ByVal value As String)
				_Password = value.Trim
			End Set
		End Property

		Public Property Server() As String
			Get
				Return _Server
			End Get
			Set(ByVal value As String)
				_Server = value.Trim
			End Set
		End Property
#End Region
	End Class
End Namespace